### ISSUE TITLE
Can't install package with React 18.1

### ISSUE THREAD

**Issue by NetoBoralli** [supasate/connected-react-router#571]
Whenever I try to install the last version of the package, using the last release of React, I receive the following error.
![image](https://user-images.githubusercontent.com/26806651/168188712-c8afcbb4-2543-4a3e-a151-2d417e04115a.png)

**Comment by melodymonyok** [comment]
Yeah, we are having this same issue as well. I haven't been able to found a work-around. Right now my only idea is to just return back to React version 17.0.0.

**Comment by awreese** [comment]
@NetoBoralli Barring any compatibility issues you could open a PR to update the project's peer dependency to include react 18. Right now though it's locked to React 16/17.

**Comment by petatemarvin26** [comment]
If you are strongly get into this try to add `--legacy-peer-deps`

**Comment by Mihai1iMob** [comment]
> @NetoBoralli Barring any compatibility issues you could open a PR to update the project's peer dependency to include react 18. Right now though it's locked to React 16/17.

PR: https://github.com/supasate/connected-react-router/pull/587
but owner seems to no longer be active as discussed in this issue: https://github.com/supasate/connected-react-router/issues/579

**Comment by awreese** [comment]
> > @NetoBoralli Barring any compatibility issues you could open a PR to update the project's peer dependency to include react 18. Right now though it's locked to React 16/17.
> 
> PR: #587 but owner seems to no longer be active as discussed in this issue: #579

Yeah, this appears to be a dead project at this point. I moved on to [redux-first-history](https://github.com/salvoravida/redux-first-history) on my newer projects and it works quite well.
